sotopia_conf/server_conf/server.gin

```diff
@@ -14,3 +15,4 @@ server.run_async_server:
     using_async=True
     tag=%TAG
     omniscient=%OMNISCIENT
+    sampler=@samplers.UniformSampler()
```

Comment:
make this a gin parameter to support customization.

Reply:
No this is not possible. You will get 
```
ValueError: No configurable matching reference '@sotopia.samplers.BaseSampler()'.
  In bindings string line 1
    SAMPLER = @sotopia.samplers.BaseSampler()
```
after setting the gin parameter

Reply:
The reason is that you have to import sotopia in the gin configuration. This is not possible from command line arguments, and should not be recommended to our users.